docs/nva-publication-channels.yaml

```diff
@@ -0,0 +1,164 @@
+openapi: 3.0.2
```

Comment:
how is this published?

Reply:
This was an attempt to do something to make openapi doc viewable - didn't quite work.... 
Now documentation i extracted from api-gateway